engine/apps/labels/models.py

```diff
@@ -105,6 +105,9 @@ class AlertReceiveChannelAssociatedLabel(AssociatedLabel):
         "alerts.AlertReceiveChannel", on_delete=models.CASCADE, related_name="labels"
     )
 
+    # If inherit is True, then the label will be passed down to alert groups
+    inherit = models.BooleanField(default=True, null=True)
```

Comment:
Wondering if `inherit` is the right word here? Maybe something more like `inheritable`, `transmit`, `spread`...? (not really sure, but it seemed confusing for me at first because this flags means that the alert group inherits the label, right? it may be a language issue though :-))

Reply:
Yes I like `inheritable` more as well

Reply:
addressed in https://github.com/grafana/oncall/pull/3307/commits/929fdcfc3e8525d61733109b4afff7c2847a9580